bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java

```diff
@@ -144,7 +144,7 @@ public void withResolver(CheckedConsumer<ResourceResolver> action) throws Except
         resolver.setCurrentItem(currentPath.get());
         try {
             action.accept(resolver.getResolver());
-        } catch (Exception ex) {
+        } catch (Throwable ex) {
```

Comment:
What's the reason for switching to `Throwable` here? In general, catching `Throwable` is a bad idea since that also catches `Error` whose javadoc essentially says not to catch it.